boards/cats_rev1Pro/src/main.c

```diff
@@ -84,13 +84,22 @@ int main(void) {
   /* Configure the system clock */
   SystemClock_Config();
 
+  MX_RTC_Init();
+
+  __HAL_RTC_WRITEPROTECTION_DISABLE(&hrtc);
+  HAL_PWR_EnableBkUpAccess();
+  uint32_t d = HAL_RTCEx_BKUPRead(&hrtc, RTC_BKP_DR0);
```

Comment:
if(HAL_RTCEx_BKUPRead(&hrtc, RTC_BKP_DR0) == BOOTLOADER_MAGIC_PATTERN)

or rename d to a better name

Reply:
Please add an explanation here, in 2 months nobody will understand this